operate/archiver/src/main/java/io/camunda/operate/archiver/ElasticsearchArchiverRepository.java

```diff
@@ -92,6 +93,7 @@ public class ElasticsearchArchiverRepository implements ArchiverRepository {
   @Autowired private Metrics metrics;
   @Autowired private RestHighLevelClient esClient;
   @Autowired private ObjectMapper objectMapper;
+  @Autowired private RichOpenSearchClient richOpenSearchClient;
```

Comment:
Opensearch clients aren't created on elasticsearch configuration, this can't be used in this class